lib/omniauth/strategies/openid_connect.rb

```diff
@@ -126,8 +126,10 @@ def callback_phase
         client.authorization_code = authorization_code
         access_token
         super
-      rescue CallbackError, ::Rack::OAuth2::Client::Error => e
-        fail!(:invalid_credentials, e)
+      rescue CallbackError => e
+        fail!(e.error, e)
+      rescue ::Rack::OAuth2::Client::Error => e
+        fail!(e.response[:error], e)
```

Comment:
Are sure sure this code works fine with new `data` argument? As far as I see from changes below you will get `ArgumentError` here because of number arguments (2 instead of 1) and I'm pretty sure `e.response[:error]` does not respond to `[]` method.

Reply:
I'm not sure, what you mean 🤔 
the second rescue is a complete different error class, my changes don't touch this part.
the first rescue works😁 we use it, as it is in a production system 😅